ci/scripts/repeat-new-tests.sh

```diff
@@ -76,9 +76,11 @@ function append_to_test_targets() {
 append_to_test_targets "repeatUnitTest" "$UNIT_TEST_CHANGES"
 append_to_test_targets "repeatIntegrationTest" "$INTEGRATION_TEST_CHANGES"
 append_to_test_targets "repeatDistributedTest" "$DISTRIBUTED_TEST_CHANGES"
-append_to_test_targets "repeatAcceptanceTest" "$ACCEPTANCE_TEST_CHANGES"
 append_to_test_targets "repeatUpgradeTest" "$UPGRADE_TEST_CHANGES"
 
+# Acceptance tests cannot currently run in parallel, so do not stress these tests
+#append_to_test_targets "repeatAcceptanceTest" "$ACCEPTANCE_TEST_CHANGES"
```

Comment:
As an aside rather than a concern, but keep in mind that `ci/` changes aren't picked up in the precheckin as a security and stability concern.  It's pretty clear what this is doing, though.